tableauserverclient/server/endpoint/jobs_endpoint.py

```diff
@@ -17,12 +18,20 @@ def get(self, job_id=None, req_options=None):
             import warnings
             warnings.warn("Jobs.get(job_id) is deprecated, update code to use Jobs.get_by_id(job_id)")
             return self.get_by_id(job_id)
+        if isinstance(job_id, RequestOptionsBase):
```

Comment:
I'm so sad. But customers will love the functionality!

Reply:
yeah once we hit 1.0, we can remove this.